src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -1385,40 +1385,6 @@ void PhaseIdealLoop::copy_assertion_predicates_to_main_loop_helper(const Predica
   }
 }
 
-// Is 'n' a node that can be found on the input chain of a Template Assertion Predicate bool (i.e. between a Template
-// Assertion Predicate If node and the OpaqueLoop* nodes)?
-static bool is_part_of_template_assertion_predicate_bool(Node* n) {
```

Comment:
Yeah, this name was not very good, "expression" would have been better than "bool".